docs/finops-hub/configure-permissions.md

```diff
@@ -0,0 +1,53 @@
+# 🛠️ Configure permissions
+
+On this page:
+
+- [✅ Requirements](#-requirements)
+- [🛠️ MCA billing accounts and billing profiles](#️-mca-billing-accounts-and-billing-profiles)
+- [🛠️ Enterprise agreement billing accounts and departments](#️-enterprise-agreement-billing-accounts-and-departments)
+- [🛠️ Subscriptions and resource groups](#️-subscriptions-and-resource-groups)
+
+---
+
+## ✅ Requirements
+
+1. [Find your Azure Active Directory tenant id](https://learn.microsoft.com/en-us/azure/azure-portal/get-subscription-tenant-id#find-your-azure-ad-tenant).
```

Comment:
I created a PR #268 to make these 2 steps as easy as looking at deployment outputs. We should use that to streamline steps.